Makefile

```diff
@@ -9,7 +9,7 @@ WITH_GOFLAGS = GOFLAGS="$(GOFLAGS)"
 ## Extra helm options
 CLUSTER_NAME ?= $(shell kubectl config view --minify -o jsonpath='{.clusters[].name}' | rev | cut -d"/" -f1 | rev | cut -d"." -f1)
 CLUSTER_ENDPOINT ?= $(shell kubectl config view --minify -o jsonpath='{.clusters[].cluster.server}')
-AWS_ACCOUNT_ID ?= $(shell aws sts get-caller-identity --output text | cut -d" " -f1)
+AWS_ACCOUNT_ID ?= $(shell aws sts get-caller-identity --query Account --output text | cut -d" " -f1)
```

Comment:
Do you need the `cut` anymore?
```
> aws sts get-caller-identity --query Account --output text
68...
```
The bug previously was `-d" "`. cut defaults to tabs IIRC, so you could've just used `cut --f1` as well.

Reply:
Good point, I removed the `cut` in a new commit.